lib/exceptions.h

```diff
@@ -48,13 +48,31 @@ class CompilerBug final : public P4CExceptionBase {
     template <typename... T>
     CompilerBug(const char* format, T... args)
             : P4CExceptionBase(format, args...)
-    { message = "COMPILER BUG:\n" + message; }
+    // \e[31m prints the text in red
+    { message = "\e[31mCOMPILER BUG\e[0m:\n" + message; }
```

Comment:
Not sure "\e5m" will convey the right message: can it blink slow enough to wait for someone to fix the bug? :)

Yeah, camel case will make it less threatening.